just: 1.40.0 -> 1.42.2

mfrw opened the pull request:
Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done
Diff: https://github.com/casey/just/compare/refs/tags/1.40.0...1.42.2
Changelog: https://github.com/casey/just/blob/1.42.1/CHANGELOG.md

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [Nixpkgs 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md) (or backporting [25.05](https://github.com/NixOS/nixpkgs/blob/master/doc/manual/release-notes/rl-2505.section.md) Nixpkgs Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
- [NixOS 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2511.section.md) (or backporting [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) NixOS Release notes)
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md), [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md), [maintainers/README.md](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md) and other contributing documentation in corresponding paths.

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

mfrw committed:
just: 1.40.0 -> 1.42.1

Diff: https://github.com/casey/just/compare/refs/tags/1.40.0...1.42.1
Changelog: https://github.com/casey/just/blob/1.42.1/CHANGELOG.md
Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>

vbgl commented:
Just in case, [1.42.2](https://github.com/casey/just/releases/tag/1.42.2) is out.

mfrw committed:
just: 1.42.1 -> 1.42.2

Diff: https://github.com/casey/just/compare/refs/tags/1.42.1...1.42.2
Changelog: https://github.com/casey/just/blob/1.42.2/CHANGELOG.md
Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>

mfrw reviewed (commented):
Supersedes:
- #424187

ryan4yin commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 425319`
Commit: `6624bc7d3831fd546d40201b223d249b3f67a73d`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 114 packages built:</summary>
  <ul>
    <li>aba</li>
    <li>catppuccin-cursors</li>
    <li>catppuccin-cursors.frappeBlue</li>
    <li>catppuccin-cursors.frappeDark</li>
    <li>catppuccin-cursors.frappeFlamingo</li>
    <li>catppuccin-cursors.frappeGreen</li>
    <li>catppuccin-cursors.frappeLavender</li>
    <li>catppuccin-cursors.frappeLight</li>
    <li>catppuccin-cursors.frappeMaroon</li>
    <li>catppuccin-cursors.frappeMauve</li>
    <li>catppuccin-cursors.frappePeach</li>
    <li>catppuccin-cursors.frappePink</li>
    <li>catppuccin-cursors.frappeRed</li>
    <li>catppuccin-cursors.frappeRosewater</li>
    <li>catppuccin-cursors.frappeSapphire</li>
    <li>catppuccin-cursors.frappeSky</li>
    <li>catppuccin-cursors.frappeTeal</li>
    <li>catppuccin-cursors.frappeYellow</li>
    <li>catppuccin-cursors.latteBlue</li>
    <li>catppuccin-cursors.latteDark</li>
    <li>catppuccin-cursors.latteFlamingo</li>
    <li>catppuccin-cursors.latteGreen</li>
    <li>catppuccin-cursors.latteLavender</li>
    <li>catppuccin-cursors.latteLight</li>
    <li>catppuccin-cursors.latteMaroon</li>
    <li>catppuccin-cursors.latteMauve</li>
    <li>catppuccin-cursors.lattePeach</li>
    <li>catppuccin-cursors.lattePink</li>
    <li>catppuccin-cursors.latteRed</li>
    <li>catppuccin-cursors.latteRosewater</li>
    <li>catppuccin-cursors.latteSapphire</li>
    <li>catppuccin-cursors.latteSky</li>
    <li>catppuccin-cursors.latteTeal</li>
    <li>catppuccin-cursors.latteYellow</li>
    <li>catppuccin-cursors.macchiatoBlue</li>
    <li>catppuccin-cursors.macchiatoDark</li>
    <li>catppuccin-cursors.macchiatoFlamingo</li>
    <li>catppuccin-cursors.macchiatoGreen</li>
    <li>catppuccin-cursors.macchiatoLavender</li>
    <li>catppuccin-cursors.macchiatoLight</li>
    <li>catppuccin-cursors.macchiatoMaroon</li>
    <li>catppuccin-cursors.macchiatoMauve</li>
    <li>catppuccin-cursors.macchiatoPeach</li>
    <li>catppuccin-cursors.macchiatoPink</li>
    <li>catppuccin-cursors.macchiatoRed</li>
    <li>catppuccin-cursors.macchiatoRosewater</li>
    <li>catppuccin-cursors.macchiatoSapphire</li>
    <li>catppuccin-cursors.macchiatoSky</li>
    <li>catppuccin-cursors.macchiatoTeal</li>
    <li>catppuccin-cursors.macchiatoYellow</li>
    <li>catppuccin-cursors.mochaBlue</li>
    <li>catppuccin-cursors.mochaDark</li>
    <li>catppuccin-cursors.mochaFlamingo</li>
    <li>catppuccin-cursors.mochaGreen</li>
    <li>catppuccin-cursors.mochaLavender</li>
    <li>catppuccin-cursors.mochaLight</li>
    <li>catppuccin-cursors.mochaMaroon</li>
    <li>catppuccin-cursors.mochaMauve</li>
    <li>catppuccin-cursors.mochaPeach</li>
    <li>catppuccin-cursors.mochaPink</li>
    <li>catppuccin-cursors.mochaRed</li>
    <li>catppuccin-cursors.mochaRosewater</li>
    <li>catppuccin-cursors.mochaSapphire</li>
    <li>catppuccin-cursors.mochaSky</li>
    <li>catppuccin-cursors.mochaTeal</li>
    <li>catppuccin-cursors.mochaYellow</li>
    <li>catppuccin-sddm</li>
    <li>celeste</li>
    <li>cosmic-applets</li>
    <li>cosmic-applibrary</li>
    <li>cosmic-bg</li>
    <li>cosmic-comp</li>
    <li>cosmic-comp.debug</li>
    <li>cosmic-design-demo</li>
    <li>cosmic-edit</li>
    <li>cosmic-ext-calculator</li>
    <li>cosmic-ext-tweaks</li>
    <li>cosmic-files</li>
    <li>cosmic-greeter</li>
    <li>cosmic-icons</li>
    <li>cosmic-idle</li>
    <li>cosmic-launcher</li>
    <li>cosmic-notifications</li>
    <li>cosmic-osd</li>
    <li>cosmic-panel</li>
    <li>cosmic-player</li>
    <li>cosmic-randr</li>
    <li>cosmic-screenshot</li>
    <li>cosmic-session</li>
    <li>cosmic-settings</li>
    <li>cosmic-store</li>
    <li>cosmic-term</li>
    <li>cosmic-workspaces-epoch</li>
    <li>cosmic-workspaces-epoch.debug</li>
    <li>dogdns</li>
    <li>dogdns.man</li>
    <li>examine</li>
    <li>find-billy</li>
    <li>forecast</li>
    <li>just</li>
    <li>just.doc</li>
    <li>just.man</li>
    <li>kabeljau</li>
    <li>libcosmicAppHook</li>
    <li>oboete</li>
    <li>onagre</li>
    <li>pop-launcher</li>
    <li>quick-webapps</li>
    <li>rofi-games</li>
    <li>ssh-openpgp-auth</li>
    <li>sshd-openpgp-auth</li>
    <li>tasks</li>
    <li>xdg-desktop-portal-cosmic</li>
    <li>xdg-desktop-portal-cosmic.debug</li>
  </ul>
</details>
